metrotranscom/doorway
# fix: hide referral section

# Pull Request Template

## Issue Overview

This PR addresses #267


## Description

This PR hides the referral section in partners on the listing edit and details pages, and removes the referral column from the listing export.

## How Can This Be Tested/Reviewed?

Check out the edit and details page of an existing listing and ensure the sections are not present. Export listings and ensure the referral column is not present.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
